code/modules/atmospherics/machinery/air_alarm/air_alarm_modes.dm

```diff
@@ -159,6 +158,25 @@ GLOBAL_LIST_INIT(air_alarm_modes, init_air_alarm_modes())
 		scrubber.set_widenet(FALSE)
 		scrubber.set_scrubbing(ATMOS_DIRECTION_SIPHONING)
 
+/datum/air_alarm_mode/supermatter
+	name = "supermatter"
+	desc = "enables scrubbers and disables internal vent limits"
+	danger = TRUE
+
+/datum/air_alarm_mode/supermatter/apply(area/applied)
+	for (var/obj/machinery/atmospherics/components/unary/vent_pump/vent as anything in applied.air_vents)
+		vent.on = TRUE
+		vent.pressure_checks = ATMOS_INTERNAL_BOUND
+		vent.internal_pressure_bound = 0
+		vent.pump_direction = ATMOS_DIRECTION_RELEASING
+		vent.update_appearance(UPDATE_ICON)
+
+	for (var/obj/machinery/atmospherics/components/unary/vent_scrubber/scrubber as anything in applied.air_scrubbers)
+		scrubber.on = TRUE
+		scrubber.filter_types = list(/datum/gas/)
+		scrubber.set_widenet(TRUE)
+		scrubber.set_scrubbing(ATMOS_DIRECTION_SCRUBBING)
```

Comment:
Wrong scrubbing define.
You dont need to set filter types

Reply:
removed the filter types